changelogs/unreleased/6013-davinci26-small.md

```diff
@@ -0,0 +1,4 @@
+## Support for Global Circuit Breaker Policy
```

Comment:
Let's bump this up to `minor` since it has a header and more than a sentence of description, so need to rename the file and update the PR label.

Reply:
fixed in 4151035aa07d44ff5b162b52dd73935da4f7cda2